js/app.js

```diff
@@ -2,27 +2,44 @@ $(document).ready(function(){
 
 var Quiz = function(questions) {
     this.questions = questions; // array of Questions.
+    this.current = 0;
 };
+//
+// $('.questionCount').text(total);
+// $('.scorePercentage').text(correct / total * 100);
```

Comment:
You will use these, but let's just keep them separate from the Quiz and Question classes.  We will use them in our `button.choice` click handlers below